src/main/java/frc/robot/commands/sequences/DeployAmpSequence.java

```diff
@@ -0,0 +1,23 @@
+package frc.robot.commands.sequences;
+
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.robot.commands.amp.DeployAmp;
+import frc.robot.commands.ramp.RampMove;
+import frc.robot.constants.Constants;
+import frc.robot.subsystems.Amp;
+import frc.robot.subsystems.Ramp;
+
+public class DeployAmpSequence extends ParallelCommandGroup {
+    public DeployAmpSequence(Ramp ramp, Amp amp) {
+        addCommands(
+            new RampMove(ramp, ()->Constants.RAMP_ANGLE),
+            new SequentialCommandGroup(
+                new WaitCommand(0.5),
```

Comment:
Is this long enough time for the ramp to get in the right position?

This is okay for now but we should make a TODO to change this once RampMove gets converted to be a synchronous method. Then this whole thing can be a SequentialCommandGroup instead of ParallelCommandGroup, without a wait().